WcaOnRails/db/migrate/20170523034604_create_post_tags.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreatePostTags < ActiveRecord::Migration[5.0]
+  def change
+    create_table :post_tags do |t|
```

Comment:
Yeah, I actually started out with that. It works in that it does create a table without an id field, but when you actually try to interact with the table, things start to fall apart. Specifically when destroying a PostTag, activerecord tries to send a sql delete where id... 

I'm not at a computer right now, but if you want to see this issue, it is easily reproduced by doing the migration you suggest and then running the tests in this PR.